T/tblis/build_tarballs.jl

```diff
@@ -14,17 +14,86 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir
 cd tblis/
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+
+for i in ./Makefile.* ./configure*; do
+
+    # Building in container forbids -march options
+    sed -i "s/-march[^ ]*//g" $i
+
+done
+
+case ${target} in
+    # Unlike stated in Wiki, 
+    # TBLIS automatically detects threading model.
+    *"x86_64"*"linux"*"gnu"*) 
+        export BLI_CONFIG=x86
+        export BLI_THREAD=openmp
+        ;;
+    *"x86_64"*"linux"*"musl"*)
+        export BLI_CONFIG=x86
+        export BLI_THREAD=pthreads
+        export CC=clang
+        export CXX=clang++
+        ;;
+    *"x86_64"*"w64"*)
+        # Windows lacks support for some instructions.
+        # Building only for AMD processors.
+        export BLI_CONFIG=amd
+        export BLI_THREAD=openmp
+        # Wrapper for posix_memalign calls.
+        echo 'aW5saW5lIGludCBwb3NpeF9tZW1hbGlnbih2b2lkICoqbWVtcHRyLCBzaXplX3QgYWxpZ25tZW50LCBzaXplX3Qgc2l6ZSkKeyAqbWVtcHRyID0gMDsgKm1lbXB0ciA9IF9hbGlnbmVkX21hbGxvYyhhbGlnbm1lbnQsIHNpemUpOyAKICAgcmV0dXJuICgqbWVtcHRyID09IDAgJiYgc2l6ZSAhPSAwKTsgfQo=' | base64 -d | cat - src/memory/aligned_allocator.hpp >> aligned_allocator_new.hpp
```

Comment:
Next time we can simply ship a more readable patch in a [`DirectorySource`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilderBase.DirectorySource) :wink:

Reply:
Thanks! I'll try to make this a `.patch` file.